backend/.env-sample

```diff
@@ -0,0 +1,7 @@
+DUMMYURL=http://localhost:8080
+POSTGRES_USER=aalto_user
+# change and uncomment or use copy-sample-env script
+#POSTGRES_PASSWORD=CHANGEME
```

Comment:
I'd personally use the same password for everyone for dev setup -- it makes things simpler, and having a different password for local envs doesn't bring any security benefits.